src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -327,6 +327,12 @@ bool layout_optimizer::can_fuse_reorder(program_node& prev, program_node& next,
          fmt_next == format::bs_fs_yx_bsv16_fsv16))
         return true;
 
+    if (next.is_type<convolution>() &&
+        fmt_prev == format::bfyx &&
+        ((fmt_next == format::b_fs_yx_fsv16 || fmt_next == format::bs_fs_yx_bsv16_fsv16) &&
+            next_output_layout.feature() >= 16 && prev_output_layout.feature() == 3))
+        return true;
```

Comment:
Does it target gen9 and gen12lp as well? If so, we need to check performance on those platforms.

Reply:
[WIP] http://ikswven004.ikor.intel.com:8080/job/release-i7-8700k-avagen9-08-dlbenchmark/10/
[PEND] http://ikswven004.ikor.intel.com:8080/job/release-i7-1165G7-avagen12-1-dlbenchmark/7/

Reply:
no graph diff on gen9 and gen12lp